content/en/profiler/enabling/ssi.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Single Step Instrumentation for Profiling
+private: true
+---
+
+## Overview
+
+Make sure you are familiar with [Single Step APM Instrumentation][1] (SSI) before proceeding.
+
+[Continuous Profiler][3] can be enabled as part of the SSI setup. Since this mode of profiler
+enablement is in private beta it is not yet integrated in the UI of the agent installation page.
+This page provides instructions on how to set it up.
+
+## Supported operating systems and environments
+
+SSI is supported only on Linux, for both `x86_64` and `arm64` (ARM v8) architectures. The beta
+version of Continuous Profiler with SSI works for host and container deployments. Kubernetes
+deployments are not supported.
+
+Beta of Continuous Profiler with SSI works with following languages on following minimum
+tracer library versions:
+
+| Language | Tracer library version |
+|----------|------------------------|
+| Java     | 1.37.0+                |
+| Node.js  | 4.39.0+, 5.15.0+       |
+| Python   | 2.11.1+                |
+
+
+## Enable Continuous Profiler with SSI
+
+Continuous Profiler can be enabled as part of the SSI setup by following these steps:
+
+1. Go to the [Agent Installation Page][2] and select one of Linux platforms or Docker.
+1. Toggle the "Enable APM Instrumentation" switch. (If there is no switch, the platform is not supported by SSI.) Toggling the switch adds the `DD_APM_INSTRUMENTATION_ENABLED=` environment variable to the installation command, configuring the installed agent to inject the tracer library into processes.
+1. Copy the installation command into a text editor.
+1. Add `DD_PROFILING_ENABLED=auto` as an additional environment variable before or after `DD_APM_INSTRUMENTATION_ENABLED` in the copied command. This turns on automatic profiler enablement for all processes.
+1. Proceed with rest of the installation instructions, using the modified installation command.
+
+## How does profiling work with SSI
+
+After the installation, all processes on the host or in the container are executed with the
+`DD_PROFILING_ENABLED=auto` environment variable. Datadog recommends using the `auto` value with SSI
+instead of the `true` value. Using `true` profiles all processes that a Datadog agent can inject a
+tracer library into. Not all processes are good candidates for profiling though. With the `auto`
+setting, the Datadog tracer library decides whether the application is a good candidate for
+profiling based on its behavior and turns on the profiler accordingly.
+
+The logic for identifying a process as a good candidate varies by language. For Java,
+all processes that are profiled as Java applications are usually deployed as a single Java process on a
+host. For Node and Python, profiler is only turned on if the application is running for more than 30
+seconds and has created at least one tracing span.
+
```

Comment:
```suggestion
SSI can also be configured to inject profiling on each and every process by using `DD_PROFILING_ENABLED=true`

Note: Datadog recommends to use `DD_PROFILING_ENABLED=auto` to avoid profiling low value processes
```